agent/php_api.c

```diff
@@ -1578,3 +1578,70 @@ PHP_FUNCTION(newrelic_get_trace_metadata) {
     nr_free(span_id);
   }
 }
+
+/*
+ * Purpose      (New Relic API) (Error Fingerprinting)
+ *              Allows a user to register a PHP callback to be invoked by the
+ *              Agent when the application encounters an error for the purpose
+ *              of returning a string for the 'error.group.name' Agent
+ *              Attribute.
+ *
+ */
+#ifdef TAGS
+void zif_newrelic_set_error_group_callback(void); /* ctags landing pad only */
+void newrelic_set_error_group_callback(void);     /* ctags landing pad only */
+#endif
+PHP_FUNCTION(newrelic_set_error_group_callback) {
+  zend_fcall_info fci;
```

Comment:
consider setting both values to `= {0}` for safety like:
```
zend_fcall_info fci = {0};
zend_fcall_info_cache fci_cache = {0};
```

here's an example of php-src using it: https://github.com/php/php-src/blob/master/sapi/embed/README.md

Reply:
Added in 1d3ea0d3da9ac486dc728a1ebd6dc9a6518f9d6f